app/routes/payments/$paymentId.tsx

```diff
@@ -23,17 +728,366 @@ export const loader: LoaderFunction = async ({ params }) => {
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        padding: '0 20px 0 20px',
+        width: '45%',
+        height: 50,
+        backgroundColor: color,
+      }}
+    >
+      <Typography>{title}</Typography>
+      <Box
+        sx={{
+          display: 'flex',
+          width: '70%',
+          justifyContent: 'space-around',
+        }}
+      >
+        <Typography noWrap>{id}</Typography>
+        <ContentCopy color="action" onClick={copyToClipBoard} />
+      </Box>
+    </Box>
+  );
+};
+
+const chipsWithColor = (
+  label: string,
+  backgroundColor: string,
+  color?: string
+) => <Chip label={label} variant="square" sx={{ backgroundColor, color }} />;
+
+const eventsJournalItem = (eventTitle: string, date: string) => (
+  <Box
+    sx={{
+      display: 'flex',
+      width: '100%',
+      padding: '10px 0 10px 0',
+      justifyContent: 'space-between',
+    }}
+  >
+    <Typography variant="body1" sx={{ display: 'flex', alignItems: 'center' }}>
+      <StopIcon sx={{ color: theme.palette.grey[500] }} />
+      {eventTitle}
+    </Typography>
+    <Typography
+      variant="footNote"
+      sx={{ display: 'flex', alignItems: 'center' }}
+    >
+      <Divider sx={{ width: '200px', 'margin-right': '20px' }} />
+      <Date timestamp={date} format="DD/MM/YYYY" />
+    </Typography>
+  </Box>
+);
+
+const dataItem = (title: string, children: ReactElement) => (
+  <Box
+    sx={{
+      display: 'flex',
+      padding: '5px 0 5px 0',
+      justifyContent: 'space-between',
+    }}
+  >
+    <Typography variant="action">{title}:</Typography>
+    <Box
+      sx={{
+        display: 'flex',
+        alignItems: 'center',
+        width: '400px',
+        justifyContent: 'flex-start',
+      }}
+    >
+      <Typography
+        variant="footNote"
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'flex-start',
+        }}
+      >
+        {children}
+      </Typography>
+    </Box>
+  </Box>
+);
+
+const titleHeader = (title: string, date: string) => (
+  <Box
+    sx={{
+      display: 'flex',
+      alignItems: 'baseline',
+    }}
+  >
+    <Typography variant="h1" sx={{ pr: '10px' }}>
+      {title}
+    </Typography>
+    <Typography
+      variant="body1"
+      sx={{ pr: '5px', color: theme.palette.neutral[600] }}
+    >
+      Date
+    </Typography>
+    <Date
+      timestamp={date}
+      format="DD/MM/YYYY"
+      color={theme.palette.neutral[600]}
+    />
+  </Box>
+);
+
+interface PaymentDetailsLoaderWrap {
+  details: PaymentDetail;
+}
+
+export default function PaymentDetails() {
+  const { details } = useLoaderData<PaymentDetailsLoaderWrap>();
   const { t } = useTranslation();
 
   return (
-    <Page id="payment" title={t('pages.payment.title')}>
-      <Typography>Id : {payment.id}</Typography>
+    <Page
+      id="payment"
+      title={titleHeader(t('pages.payment.title'), details.createdAt)}
+      onClick={() => null}
```

Comment:
maybe check with clem for the redirection link, or if we don't have one yet, maybe we should make it disabled or simply to displaying it